<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../OneDrive/Desktop/logo isquare.png</normaloff>../../OneDrive/Desktop/logo isquare.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit_Username">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Login">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;RESET</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SignUp">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Forgot_Password">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Forgot Password</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Logos/AppLogo128x128.ico</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Username</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
